components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1/src/main/java/org/wso2/carbon/apimgt/rest/api/publisher/v1/impl/ApisApiServiceImpl.java

```diff
@@ -3852,6 +3860,17 @@ public String getMediationNameFromConfig(String config) {
         return null;
     }
 
+    /**
+     * Check the existence of the mediation policy
+     * @param mediationResourcePath mediation config content
+     *
+     */
+    public void checkMediationPolicy(APIProvider apiProvider,String mediationResourcePath) throws APIManagementException {
```

Comment:
Let's fix the formatting issue in this method. Need a space after the comma in the list of parameters.

Reply:
Fix with 5872157